platform/configure/config.mdx

```diff
@@ -79,79 +83,83 @@ helm upgrade loft vcluster-platform --install \
   --version $PLATFORM_VERSION
 ```
 
-### Custom HTTP Headers
-
-vCluster Platform allows you to add custom HTTP headers to all requests that are sent to the vCluster Platform API. This can be useful if you want to add custom headers to all requests, such as `X-Frame-Options` or `X-XSS-Protection`.
+<!-- vale off -->
+### Custom HTTP headers
+<!-- vale on -->
+The platform allows you to add custom HTTP headers to all requests that are sent to the platform API. This can be useful if you want to add custom headers to all requests, such as `X-Frame-Options` or `X-XSS-Protection`.
 
 To add custom HTTP headers, use the following configuration:
 
-```yaml
+```yaml title="Custom HTTP headers configuration"
 config:
   auth:
     customHttpHeaders:
       X-Frame-Options: SAMEORIGIN
       X-XSS-Protection: 1; mode=block
 ```
 
-### Setting additional certificate authority for Platform
-If you want to have **all** your virtual clusters managed by the vCluster Platform to use a specific certificate authority to establish the connection between the virtual cluster and the platform, you can set it as a configuration option on the vCluster Platform instead of configuring it on each individual virtual cluster.
-This will enable secure communication between your Virtual Clusters and the vCluster Platform.
-As a pre-requisite, you have to [configure TLS for the vCluster Platform](/platform/configure/domain#configure-tls) first.
-```yaml
+### Setting additional certificate authority for platform
+
+If you want to have **all** your virtual clusters managed by the platform use a
+specific certificate authority to establish the connection between the virtual
+cluster and the platform, you can set it as a configuration option on the
+platform instead of configuring it on each individual virtual cluster. This
+enables secure communication between your virtual clusters and the platform.
+
+As a prerequisite, you have to [configure TLS for the platform](/platform/configure/domain#configure-tls) first.
```

Comment:
This feels buried in the docs and something we should make more obvious if it's a pre-req.